_posts/en/newsletters/2022-05-25-newsletter.md

```diff
@@ -0,0 +1,193 @@
+  peer requests the `pckginfo1` message, they can use the info in the
+  message to determine whether they really are interested in the package
+  and to learn the wtxids of any transactions they need to download in
+  order to validate the high feerate child.  The actually transactions can
+  be requested with a `getpckgtxns` message and received in a `pckgtxins`
+  message.
+
+  Although the [draft BIP][bip-package-relay] focuses on just the
+  protocol, Zhao's email provides additional context, briefly describes
+  alternative designs that were considered by found deficient, and links
+  to a [presentation][zhao preso] with additional details.
+
+- **Miner extractable value discussion:** developer /dev/fd0
+  [posted][fd0 ctv9] to the Bitcoin-Dev mailing list a summary of the
+  [ninth IRC meeting][ctv9] about [OP_CHECKTEMPLATEVERIFY][topic
+  op_checktemplateverify] (CTV).  Among other topics discussed during
+  that meeting, Jeremy Rubin listed several concerns he's heard people
+  express about recursive [covenants][topic covenants] (which CTV does
+  not enable).  One of those concerns was creating Miner Extractable
+  Value (MEV) significantly beyond the amount available from running a
+  simple transaction selection algorithm such as that provided by
+  Bitcoin Core.
+
+    MEV has become a particular concern in Ethereum and related
+    protocols where the use of public onchain trading protocols allows
+    miners to frontrun trades.  For example, imagine the following two
+    unconfirmed transactions both being available to mine in the next
+    block:
+
+    * Alice sells *x* to Bob for 1 ETH
+    * Bob sells *x* to Carol for 2 ETH (Bob earns 1 ETH in profit)
+
+    <br>If those two exchanges are conducted using a public onchain trading
+    protocol, a miner can cut Bob out of the transaction.  For example:
+
+    * Alice sells *x* to Miner Mallory for 1 ETH
+    * Miner Mallory sells *x* to Carol for 2 ETH (Mallory earns 1 ETH in
+      profit; Bob earns nothing)
+
+    <br>Obviously this is a problem for Bob, but it also creates several
+    problems for the network.  The first problem is that, in the above
+    example, Miner Mallory needs to have 1 ETH in a wallet in order to
+    exploit the situation.  Every other miner attempting to mine the
+    next block also needs to have their own 1 ETH.  That wallet value is
+    independent of each miner's hashrate, so two miners can join
+    together to halve the amount of money they each need to store in
+    their wallets for capturing MEV---producing an economy of scale that
+    encourages centralization of mining and which can leave the network
+    more vulnerable to transaction censorship.  A [report][bitmex
+    flashbots] by BitMex Research claims that a centralized service that
+    brokers these types of MEV transactions was being used by 90% of
+    Ethereum hashrate at the time the report was written.  To obtain the
```

Comment:
Flashbots is doing something else actually, it's not about capital. It's about the "searchers" and strategy makers coordinating with the miners to get paid even if they don't mine.